src/main/java/org/candlepin/insights/controller/InventoryController.java

```diff
@@ -117,7 +118,13 @@ public ConduitFacts getFactsFromConsumer(Consumer consumer) {
     private void extractHardwareFacts(Map<String, String> pinheadFacts, ConduitFacts facts) {
         String systemUuid = pinheadFacts.get(DMI_SYSTEM_UUID);
         if (!isEmpty(systemUuid)) {
-            facts.setBiosUuid(systemUuid);
+            if (systemUuid.matches(UUID_REGEX)) {
+                facts.setBiosUuid(systemUuid);
+            }
+            else {
+                log.warn(String.format("Consumer %s in org %s has unparseable BIOS uuid: %s",
```

Comment:
Instead of this, use the slf4j placeholder style of formatting messages:

```java
log.warn("Consumer {} in org {} has unparseable BIOS uuid: {}",
    facts.getSubscriptionManagerId(), facts.getOrgId(), systemUuid);
```

That should resolve the SonarCloud check.